packages/theme-toolkit/package.json

```diff
@@ -20,6 +20,8 @@
     "react": "16.14.0"
   },
   "devDependencies": {
+    "@gemeente-denhaag/components-react": "0.1.1-alpha.267",
+    "@gemeente-denhaag/file": "0.1.1-alpha.1",
```

Comment:
Ik denk dat deze niet meer nodig is als denhaag wordt geupdate want die file import zit in die nieuwste versie daar netjes in

Reply:
Aangepast!